frontend/views/pages/Join.vue

```diff
@@ -86,7 +75,7 @@ export default {
     pageStatus: {
       get () { return this.ephemeral.pageStatus },
       set (status) {
-        const posibleStatus = ['LOADING', 'WELCOME', 'SIGNING', 'LOGGING', 'EXPIRED', 'INVALID']
```

Comment:
What happened to expired links?

Reply:
The layout is the same for expired an invalid links to it was a redundant status.